docs/accessing-clusters.md

```diff
@@ -0,0 +1,30 @@
+
+## Accessing clusters running on AWS
+Before proceeding, please ensure you have the following tools installed:
+* awscli
+* pulumi
+* kubectl
+
+### Steps
+1. Login using the following commands
+```shell
+cd src/projects/aws-eks/
+pulumi login s3://ai-pulumi-stacks
+```
+2. You can now view the available clusters using,
+```shell
+pulumi stack ls
+```
+3. To configure kubectl to work with a specific cluster,
+```shell
+export PULUMI_CONFIG_PASSPHRASE=******* 
+pulumi stack select <replace-with-cluster-name> # check result from step 2
+pulumi stack output kubeconfig > kubeconfig
+export KUBECONFIG=`pwd`/kubeconfig
+pulumi stack output awsProfileSnippet >> ~/.aws/config
```

Comment:
the result of this command in ~/.aws/config file is
```
[<replace-with-cluster-name>]
region = ...
output = ...
role_arn = ...
source_profile = default
```

we should notice users to change that file a little bit in order to run steps afterward successfully
```
[profile <replace-with-cluster-name>]
region = ...
output = ...
role_arn = ...
source_profile = <profile-to-use-in-aws-if-user-have-more-than-1-aws-profile>
```

I have 2 errors when running export AWS_PROFILE and kubectl commands if we don't edit that aws config file:
- The config profile (<replace-with-cluster-name>) could not be found
- An error occurred (InvalidClientTokenId) when calling the AssumeRole operation: The security token included in the request is invalid.

Reply:
There is a PR from @milliondreams  where this is fixed - https://github.com/Aitomatic/melody/pull/13/files#diff-9eb25afdfb66ec7e05b6a7c9b60e2712f0f517ae91ba83bfde28100b15310a12R127